src/src/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -136,28 +148,68 @@ public boolean canUseLocalCache() {
 
         return false;
     }
-
+        
     /**
-     * App needs to give permission to AccountManager to use broker.
+     * To verify if App gives permissions to AccountManager to use broker.
+     * 
+     * @throws DeveloperAuthenticationException
+     * @return boolean If the GET_ACCOUNTS, MANAGE_ACCOUNTS, USE_CREDENTIALS permissions are granted in the Manifest.xml
      */
     private boolean verifyManifestPermissions() {
```

Comment:
I added the throws after the method name anyway... However, we do not need to. https://docs.oracle.com/javase/tutorial/essential/exceptions/throwing.html